content/200-orm/100-prisma-schema/10-overview/04-location.mdx

```diff
@@ -82,6 +82,12 @@ datasource db {
 }
 ```
 
+:::info
+
+When first updating to the `prismaSchemaFolder` preview feature, your IDE may not recognize the new format immediately. Be sure to restart your IDE to confirm you have the latest version of the Prisma VS Code Extension installed.
```

Comment:
What is this based on? That should not be the case. If this really happens, we should treat it as a bug and fix it.

Reply:
Sorry, let me clarify
```suggestion
When first updating to Prisma ORM 5.15.0 or later, your IDE may not recognize the new multi-file format immediately. Be sure to restart your IDE to confirm you have the latest version of the Prisma VS Code Extension installed.
```

Reply:
Ah ok, that is less drastic.

Is restarting really enough? Not better "check your extensions that you have at least version 5.15.0 of the Prisma VS Code Extension isntalled" or something like that?

Reply:
We already have instructions for updating the VSCode extension. This block was specifically for some confusion that folks in Discord had. So just covering more bases.